meta-intel-edison-distro/conf/distro/poky-edison.conf

```diff
@@ -6,11 +6,13 @@ PREFERRED_PROVIDER_zlib ?= "zlib"
 PREFERRED_PROVIDER_zlib-native ?= "zlib-native"
 PREFERRED_PROVIDER_nativesdk-zlib ?= "nativesdk-zlib"
 # LTS kernel
-#PREFERRED_VERSION_linux-yocto = "5.15.25"
+PREFERRED_VERSION_linux-yocto = "5.15.25"
```

Comment:
I'm a bit lost here, shouldn't we bump the minor from 25 to 81 or so?

Reply:
That is correct. I bump the version in a later patch.